### PR TITLE
Mybranch

### PR DESCRIPTION
**Pull request by Acerpo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2171,3 +2171,9 @@ My name is Kateryna and I'm happy to join this community!
 
 My name is [Dima](https://github.com/dimonlakhin). I decided to switch to a new profession. So I am glad to join the community. I hope to get your help and support.
 ![This is my own cat Shaya](./assets/images/dimon89.jpg)
+
+## Hi, everyone!
+My name is [Andrii Shchur](https://github.com/Acerpo).
```
Comment (artem-trubin) [inline README.md:2176]: I see empty spaces at the end of several of your lines. Please remove them as it is a bad habit. You can set your text editor (VSCode for example) to remove them automatically the moment you save your file.

File: README.md [hunk 2]
```diff
@@ -2171,3 +2171,9 @@ My name is Kateryna and I'm happy to join this community!
 
 My name is [Dima](https://github.com/dimonlakhin). I decided to switch to a new profession. So I am glad to join the community. I hope to get your help and support.
 ![This is my own cat Shaya](./assets/images/dimon89.jpg)
+
+## Hi, everyone!
+My name is [Andrii Shchur](https://github.com/Acerpo). 
+Last time I did not start correctly, so we start from the beginning on 07/19/2022
+I want to learn frontend! 
+![Cat](./assets/images/mycat_A.jpg)
```
Comment (artem-trubin) [inline README.md:2179]: As you see, Markdown palced the cat on the "same line" as your text, in Markdown you need a full empty line most blocks to consider them "on different lines". Putting an empty line before your picture will move it under the text nicely.

**Review by artem-trubin — COMMENTED** [review]
What a nice can you have!
Let's make some changes to make your PR better.

Choose a better name for the PR. Generally you would like to make more descriptive names to you branches, commits and PRs, so other people can instantly see what is what. Since your commits will be squashed and branch will be later deleted, they are not that relevant right now. But PR is, as it will be in history of this repo. Call it something like "Added a new can" or "Acerpo's new entry".

File: README.md (hunk 1 above)
Comment (Acerpo) [reply README.md:2176]: I do not understand what you're saying. How do you see empty spaces?

Installed Prettier, but I don't understand what has changed.

Comment (artem-trubin) [reply README.md:2176]: Hi, I'm very sorry, must've caught a bug or something, I don't see any now, this was on my side. All fine with your line endings :)

**Comment by artem-trubin** [comment]
If you opened another PR on the task, delete the old one please (this one).

**Comment by boxdxnx** [comment]
@Acerpo Resolve conflicts, please.

**Comment by boxdxnx** [comment]
One more "duplication". 
Closing it 😸

**Pull request closed without merging**